.github/workflows/release.yml

```diff
@@ -100,6 +100,35 @@ jobs:
                   else
                     false
                   fi
+    check-links:
+        runs-on: ubuntu-latest
+        needs: ["set-tags"]
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+              with:
+                  ref: ${{ needs.set-tags.outputs.git_ref }}
+            - uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
This action is no longer supported IIUC. The creator recommends this improved action based on LinkSpecter tool: https://github.com/UmbrellaDocs/action-linkspector

Reply:
Ok, I will just remove it then